### PR TITLE
Added VLM bindings and a Python sample.

### PR DESCRIPTION
**Pull request by popovaan**
- Added VLM bindings.
- Added Python VLM chat sample.
- Added initialization of performance metrics with zeros.
Tickets: CVS-153174, CVS-153173, CVS-153626

PR to miniCPM-V-2_6 branch: https://github.com/Wovchena/openvino.genai-public/pull/62

### CODE REVIEW

**Review by Wovchena — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by Wovchena** [comment]
There are going to be merge conflicts

**Comment by Wovchena** [comment]
build_jenkins

**Comment by Wovchena** [comment]
Please resolve the conflicts

File: .github/workflows/causal_lm_cpp.yml [hunk 1]
```diff
@@ -707,7 +708,9 @@ jobs:
           python -m pip install --upgrade-strategy eager -r ./samples/requirements.txt --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
           python ./samples/cpp/visual_language_chat/export_MiniCPM-V-2_6.py ./miniCPM-V-2_6/
           wget https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/d5fbbd1a-d484-415c-88cb-9986625b7b11
-      - run: >
+          
+      - name: Run chat chat sample
+        run: |
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:713]: @Wovchena should it be?
```suggestion
        run: >
```

**Comment by Wovchena** [comment]
build_jenkins

**Pull request merged**